week-2/14-align-self/readme.md

```diff
@@ -0,0 +1,13 @@
+# Align self
+
+With `align-items` you move all the items at once (on the **non-main axis**).
```

Comment:
Have we described the axis before this exercise? Maybe we need an exercise specifically on the axis that occurs before this?

Reply:
I was assuming we will describe it in class - I can add an exercise in that case. Something to indicate how `justify-content` and `align-items` are linked to `flex-direction`

Reply:
:+1: